src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -150,26 +264,38 @@ public static synchronized ArrayList getFifoNextSpanDataAllGroups(boolean startN
    * @param includeTagGroupD if tag group D data should be included
    * @return historical log data
    * @throws IOException if unable to read or write files
+   * @throws TimeTrackerUnrecoverableException if both time tracking files are corrupted
+   * @throws CorruptedTimeTrackerException if the primary time tracking file is corrupted
    */
   public static synchronized ArrayList getFifoNextSpanData(
       boolean startNewTimeTracker,
```

Comment:
Doesn't appear that `startNewTimeTracker` is used anymore. I think it should either still be used to force new time trackers to be created, or be removed. I lean more towards still handling that boolean, because I do see a use case where a connector could force a time tracker reset. In the past, I have told customers how to shut off a connector, and manually delete the tracker files so they are recreated at current time.

Reply:
startNewTimeTracker is still used on line 297. I was thinking that behavior of passing startNewTimeTracker as true rather then rely on exceptions in the case the files don't exist makes more sense. The exception handling is really just for error's.

Reply:
missed that